src/Components/home/Calendar.jsx

```diff
@@ -28,3 +30,12 @@ export default function Calendar() {
     />
   );
 }
+
+/*
+[
+        { title: 'event 1', start: '2020-05-01', end: '2020-05-04' },
+        { title: 'event 2', start: '2020-05-03', end: '2020-05-06' },
+        { title: 'event 3', start: '2020-05-03', end: '2020-05-06' },
+        { title: 'event 4', start: '2020-05-03', end: '2020-05-06' },
+      ]
+      */
```

Comment:
It's usually best practice to delete commented code, since we can use git history to retrieve it later.

Reply:
Makes sense!